iGCS/Managers/GCSSpeechManager.h

```diff
@@ -0,0 +1,14 @@
+//
+//  GCSSpeechManager.h
+//  iGCS
+//
+//  Created by Andrew Brown on 1/12/15.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@interface GCSSpeechManager : NSObject
++ (instancetype) sharedInstance;
+- (void) flyToPosition;
```

Comment:
Very minor nit - this looked more like a property name at first glance. "sayFlyingToPosition"?